styles.css

```diff
@@ -0,0 +1,83 @@
+.nav-bar {
+  display: flex;
+  align-items: baseline;
+  gap: 10px;
+  width: 450px;
+  padding-top: 20px;
+  font-style: normal;
+  justify-content: flex-end;
+  color: blue;
+}
+
+a {
+  font-style: normal;
+  margin-right: auto;
+  margin-left: 2rem;
+  font-weight: bolder;
+}
+
+p {
+  color: #172f58;
+  align-content: space-around;
+  width: 400px;
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+}
+
+body {
+  background-color: white;
+  font-family: 'Poppins', sans-serif;
+}
+
+.headline-container {
+  @import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,500;1,600;1,700&display=swap');
+
+  display: flex;
+  align-content: flex-start;
+  margin: 50px;
+  background-image: url(./images/background.png);
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] You did well adding the background image as suggested by the previous reviewer. However, its margin is a bit too much making your design differs a bit from the figma design.

Your Design | Figma Design |
:---------------:|:-------------------:|
![a1](https://user-images.githubusercontent.com/39523312/164836553-3c5c4ae5-ba05-4e69-a84d-6f6e72790a99.png)| ![a2](https://user-images.githubusercontent.com/39523312/164836557-3fb0651c-82e5-4aeb-b1fb-001f713bca6c.png)